scripts/install-helpers/baremetal-coco/intel-dcap/pccs.yaml.in

```diff
@@ -81,46 +39,42 @@ spec:
         trustedservices.intel.com/cache: pccs
     spec:
       tolerations:
-      - effect: NoSchedule
-        key: node-role.kubernetes.io/master
-        operator: Exists
+        - effect: NoSchedule
+          key: node-role.kubernetes.io/master
+          operator: Exists
       nodeSelector:
         kubernetes.io/hostname: ${PCCS_NODE}
-      initContainers:
-       - name: init-seclabel
-         image: registry.access.redhat.com/ubi9/ubi:latest
-         command: ["sh", "-c", "chcon -Rt container_file_t /var/cache/pccs"]
-         volumeMounts:
-         - name: host-database
-           mountPath: /var/cache/pccs
-         securityContext:
-           runAsUser: 0
-           runAsGroup: 0
-           privileged: true  # Required for chcon to work on host files
       containers:
         - name: pccs
-          image: quay.io/openshift_sandboxed_containers/dcap/pccs:0.2.0
+          image: quay.io/pproskur/openshift/pccs:0.2.4 # TODO WAIT FOR RELEASE
```

Comment:
Should this be changed to the newly built images from konflux ?

Reply:
This PR will work with merged: https://github.com/openshift/confidential-compute-artifacts/pull/24

Reply:
@pawelpros can you try with this quay.io/redhat-user-workloads/ose-osc-tenant/osc-pccs:osc-pccs-on-push-d7xph-build-images ?